.github/workflows/firebase_test_lab.yml

```diff
@@ -73,5 +73,5 @@ jobs:
             --device model=redfin,version=30,locale=en,orientation=portrait \
             --directories-to-pull /sdcard/Download \
             --results-bucket gs://macrobenchmark-results \
-            --environment-variables clearPackageData=true,additionalTestOutputDir=/sdcard/Download,no-isolated-storage=true \
+            --environment-variables clearPackageData=true,additionalTestOutputDir=/sdcard/Download,no-isolated-storage=true,androidx.benchmark.enabledRules=Macrobenchmark,numShards=3 \
```

Comment:
@tikurahul the tests are failing due to the timeout. Should I increase timeout to ~50min? Or I'm trying numShards to paralelize the tests?